src/common/transformations/tests/common_optimizations/group_normalization_fusion_tests.cpp

```diff
@@ -0,0 +1,570 @@
+    GroupNormalizationFusionTestsFixture_i32::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_i64, GroupNormalizationFusionTests_i64) {
+    GroupNormalizationFusionTestsFixture_i64::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f8e4m3, GroupNormalizationFusionTests_f8e4m3) {
+    GroupNormalizationFusionTestsFixture_f8e4m3::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f8e5m2, GroupNormalizationFusionTests_f8e5m2) {
+    GroupNormalizationFusionTestsFixture_f8e5m2::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f4e2m1, GroupNormalizationFusionTests_f4e2m1) {
+    GroupNormalizationFusionTestsFixture_f4e2m1::TestBody();
+}
+
+TEST_P(GroupNormalizationFusionTestsFixture_f8e8m0, GroupNormalizationFusionTests_f8e8m0) {
+    GroupNormalizationFusionTestsFixture_f8e8m0::TestBody();
+}
+
+using RawValuesContainer = std::tuple<PartialShape, Shape, Shape, Shape, Shape, unsigned long long, float>;
+using ValuesContainerWithPositiveTestFlag =
+    std::tuple<bool, PartialShape, Shape, Shape, Shape, Shape, unsigned long long, float>;
+
+std::vector<RawValuesContainer> valid_vals = {
+    std::make_tuple(Shape{1, 320}, Shape{}, Shape{}, Shape{320}, Shape{320}, 1, 1e-5f),
+    std::make_tuple(Shape{1, 320, 2, 2},
+                    Shape{1, 1, 1},
+                    Shape{1, 1, 1},
+                    Shape{320, 1, 1},
+                    Shape{1, 320, 1, 1},
+                    1,
+                    1e-5f),
+    std::make_tuple(Shape{1, 320, 2, 2},
+                    Shape{1, 320, 1},
+                    Shape{1, 320, 1},
+                    Shape{320, 1, 1},
+                    Shape{320, 1, 1},
+                    320,
+                    1e-5f),
+    std::make_tuple(PartialShape{Dimension::dynamic(), 320, Dimension::dynamic(), Dimension::dynamic()},
+                    Shape{1, 320, 1},
+                    Shape{1, 320, 1},
+                    Shape{320, 1, 1},
+                    Shape{320, 1, 1},
+                    320,
+                    1e-5f),
+    std::make_tuple(PartialShape{Dimension::dynamic(), 320},
+                    Shape{32, 1},
+                    Shape{32, 1},
+                    Shape{320},
+                    Shape{320},
+                    32,
+                    1e-5f),
+    std::make_tuple(PartialShape{1, 320, Dimension::dynamic()},
+                    Shape{32, 1},
+                    Shape{32, 1},
+                    Shape{320, 1},
+                    Shape{320, 1},
+                    32,
+                    1e-5f),
+    std::make_tuple(PartialShape{1, 320, 2, Dimension::dynamic()},
+                    Shape{1, 32, 1},
+                    Shape{1, 32, 1},
+                    Shape{320, 1, 1},
+                    Shape{320, 1, 1},
+                    32,
+                    1e-5f),
+    std::make_tuple(Shape{2, 320, 4, 8}, Shape{}, Shape{}, Shape{320, 1, 1}, Shape{1, 320, 1, 1}, 32, 1e-5f),
```

Comment:
```suggestion
    std::make_tuple(PartialShape{2, 320, 4, 8}, Shape{}, Shape{}, Shape{320, 1, 1}, Shape{1, 320, 1, 1}, 32, 1e-5f),
```
there is no need to convert it from ov::Shape

Reply:
Done